### ISSUE TITLE
x/build/env/linux-arm64/packet: all builders down

### ISSUE THREAD

**Issue by dmitshur** [golang/go#32543]
According to https://farmer.golang.org/#health, all the Packet linux/arm64 machines are missing (as of 3 hours ago):

<img src="https://user-images.githubusercontent.com/1924134/59236300-818e9700-8bc3-11e9-83d1-3a60d1499f1f.png" width="337px" height="358px">

I will be looking into this soon.

**Comment by dmitshur** [comment]
`ssh`ing into the server times out:

```
$ ssh -v -i ~/.ssh/id_ed25519_golang1 root@147.75.74.194
[...]
ssh: connect to host 147.75.74.194 port 22: Operation timed out
```

Pinging it times out too:

```
$ ping 147.75.74.194
PING 147.75.74.194 (147.75.74.194): 56 data bytes
Request timeout for icmp_seq 0
Request timeout for icmp_seq 1
Request timeout for icmp_seq 2
Request timeout for icmp_seq 3
```

This appears to be very similar to the last time packet builders had a problem in issue #31112.

**Comment by dmitshur** [comment]
I've restarted the server and that fixed the issue (just like in #31112).

It took a few minutes for it to fully restart, but then it became responsive to pings, ssh, and all the packet builders are up and connected now:

<img src="https://user-images.githubusercontent.com/1924134/59237016-94569b00-8bc6-11e9-8407-2559beae39ec.png" width="276px" height="18px">
